uiowa/uiowa
# [ DX ] appSearch

# How to test

1. Pull branch
2. Open the top level index.html and check how it works
3. Approve
4. Or else
